e2e/index.ts

```diff
@@ -1,42 +1,46 @@
-import { expect } from "chai";
 import * as dotenv from "dotenv";
-import { before, describe, it } from "mocha";
+import { before, describe } from "mocha";
+import { generateToken } from "../src/cryptography/create-token";
 import { AppDataSource } from "../src/data-source";
 import { User } from "../src/entity/User";
-import {
-  CreateUserInput,
-  createUserMutation,
-} from "../src/mutation/create-user.use-case";
+import { CreateUserInput } from "../src/mutation/create-user.use-case";
 import { setup } from "../src/setup";
+import { saveUserToDB } from "./save-user-to-db";
+import { CreateUserTest } from "./scripts/create-user-script";
+import { GetUserTest } from "./scripts/get-user-script";
+import { LoginTest } from "./scripts/login-script";
 
-const TEST_VARIABLE: CreateUserInput = {
+export const TEST_VARIABLE: CreateUserInput = {
   name: "john-doe",
   email: "john-doe@email.com",
-  password: "p4ssw0rd",
+  password: "passw0rd",
   birthDate: "00-00-0000",
 };
 
 describe("Test", async () => {
+  let token: string;
+  let TestUser: User;
+
   before(async () => {
     dotenv.config({ path: __dirname + "/../test.env" });
     await setup();
-    await AppDataSource.manager.clear(User);
   });
 
-  it("Should Create User", async () => {
-    const response = await createUserMutation(TEST_VARIABLE);
-    const { id, ...otherFields } = response.data.data.createUser;
-    const { password, ...inputOtherFields } = TEST_VARIABLE;
+  beforeEach(async () => {
+    await AppDataSource.manager.clear(User);
+    TestUser = await saveUserToDB();
+    token = generateToken(TestUser, "secretKey", 1200);
+  });
 
-    expect(otherFields).to.be.deep.equal(inputOtherFields);
+  describe("Login Test", async () => {
+    await LoginTest();
+  });
 
-    expect(id).to.be.a("number");
+  describe("CreateUser Test", async () => {
+    await CreateUserTest();
+  });
```

Comment:
Vc pode dar só um require no arquivo que ele faz isso que vc pensou importando a chamando a função:

```suggestion
  require('./scripts/create-user-script');
```

Reply:
E lá no arquivo vc pode remover a função e deixar direto o describe, que ele roda os testes normalmente.